build.sbt

```diff
@@ -4879,6 +4881,17 @@ lazy val cleanPolyglotRoot = taskKey[Unit](
   "Helper task that prepares polyglot directory of a stdlib component"
 )
 
+lazy val `opencv-thin` = project
+  .in(file("std-bits") / "opencv-thin")
+  .enablePlugins(JarExtractPlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.openpnp" % "opencv" % opencvVersion
+    ),
+    inputJar := "org.openpnp" % "opencv" % opencvVersion,
+    jarExtractor := JarExtract.openCVExtractor
```

Comment:
- good for now, but I'd like to see the (declarative) configuration here
- why not:
```
polyglotLib["amd64"] := Seq(
  ".*/windows/x86_64/.*.dll", 
  ".*/linux/x86_64/.*.so", 
  ".*/osx/x86_64/.*.dylib"
)
polyglotLib["aarch64"] := Seq(".*/osx/aarch64/.*.dylib")
```
- either as regex or glob patter
- those `polyglotLib` resources would automatically be removed from the "thin JAR"
- we could have `skipResources := Seq(....)` to drop some resources
- the rest would get into the thin JAR